### PR TITLE
Making the "My Goals +"button take the user to th 'my Goals' page

### PR DESCRIPTION
**Pull request by GithinjiHans**

### CODE REVIEW

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 1]
```diff
@@ -147,31 +139,14 @@ export const UserChoiceDashboard = () => {
...
-                                ? "dashboard-add-btn-dark"
-                                : "dashboard-add-btn-light"
-                        }
-                    >
-                        {darkModeStatus ? (
-                            <img
-                                src={AddIconDark}
-                                alt="Add Icon"
-                                className="add-icon"
-                            />
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:166]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 2]
```diff
@@ -190,28 +165,7 @@ export const UserChoiceDashboard = () => {
                     >
                         {truncateContent(t("zinzen"))}
                     </Button>
-                    <Button
-                        variant={darkModeStatus ? "dark-purple" : "purple"}
-                        className={
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:195]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 3]
```diff
@@ -190,28 +165,7 @@ export const UserChoiceDashboard = () => {
                     >
                         {truncateContent(t("zinzen"))}
                     </Button>
-                    <Button
-                        variant={darkModeStatus ? "dark-purple" : "purple"}
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:194]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 4]
```diff
@@ -190,28 +165,7 @@ export const UserChoiceDashboard = () => {
                     >
                         {truncateContent(t("zinzen"))}
                     </Button>
-                    <Button
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:193]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:193]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:193]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:193]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:193]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx (hunk 3 above)
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:194]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:194]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:194]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:194]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:194]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx (hunk 2 above)
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:195]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx (hunk 3 above)
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:194]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx (hunk 2 above)
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:195]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:195]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 5]
```diff
@@ -190,28 +165,7 @@ export const UserChoiceDashboard = () => {
...
-                        variant={darkModeStatus ? "dark-purple" : "purple"}
-                        className={
-                            darkModeStatus
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:196]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:196]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 6]
```diff
@@ -190,28 +165,7 @@ export const UserChoiceDashboard = () => {
...
-                                ? "dashboard-add-btn-dark"
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:197]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 7]
```diff
@@ -190,28 +165,7 @@ export const UserChoiceDashboard = () => {
...
-                                : "dashboard-add-btn-light"
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:198]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:198]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 8]
```diff
@@ -190,28 +165,7 @@ export const UserChoiceDashboard = () => {
...
-                                alt="Add Icon"
-                                className="add-icon"
-                            />
-                        ) : (
-                            <img
-                                src={AddIconLight}
-                                alt="Add Icon"
-                                className="add-icon"
-                            />
-                        )}
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:213]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 9]
```diff
@@ -190,28 +165,7 @@ export const UserChoiceDashboard = () => {
...
-                    </Button>
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:214]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:214]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:214]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:214]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:214]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 10]
```diff
@@ -190,28 +165,7 @@ export const UserChoiceDashboard = () => {
                     >
                         {truncateContent(t("zinzen"))}
                     </Button>
-                    <Button
-                        variant={darkModeStatus ? "dark-purple" : "purple"}
-                        className={
-                            darkModeStatus
-                                ? "dashboard-add-btn-dark"
-                                : "dashboard-add-btn-light"
-                        }
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:199]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:199]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:199]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 11]
```diff
@@ -190,28 +165,7 @@ export const UserChoiceDashboard = () => {
...
-                    >
-                        {darkModeStatus ? (
-                            <img
-                                src={AddIconDark}
-                                alt="Add Icon"
-                                className="add-icon"
-                            />
-                        ) : (
-                            <img
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:208]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 12]
```diff
@@ -190,28 +165,7 @@ export const UserChoiceDashboard = () => {
...
-                                src={AddIconLight}
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:209]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:209]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx (hunk 8 above)
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:213]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx (hunk 9 above)
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:214]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:214]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:214]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:214]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:214]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:214]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx (hunk 5 above)
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:196]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx (hunk 8 above)
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:213]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx (hunk 7 above)
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:198]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:198]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx (hunk 10 above)
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:199]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 13]
```diff
@@ -190,28 +165,7 @@ export const UserChoiceDashboard = () => {
...
                         {truncateContent(t("zinzen"))}
                     </Button>
-                    <Button
-                        variant={darkModeStatus ? "dark-purple" : "purple"}
-                        className={
-                            darkModeStatus
-                                ? "dashboard-add-btn-dark"
-                                : "dashboard-add-btn-light"
-                        }
-                    >
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:200]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 14]
```diff
@@ -190,28 +165,7 @@ export const UserChoiceDashboard = () => {
...
-                        {darkModeStatus ? (
-                            <img
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:202]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx [hunk 15]
```diff
@@ -190,28 +165,7 @@ export const UserChoiceDashboard = () => {
...
                     </Button>
-                    <Button
-                        variant={darkModeStatus ? "dark-purple" : "purple"}
-                        className={
-                            darkModeStatus
-                                ? "dashboard-add-btn-dark"
-                                : "dashboard-add-btn-light"
-                        }
-                    >
-                        {darkModeStatus ? (
```
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:201]: Removing the + button

File: src/components/dashboard/UserChoiceDashboard.jsx (hunk 9 above)
Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:214]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:214]: Removing the + button

Comment (GithinjiHans) [inline src/components/dashboard/UserChoiceDashboard.jsx:214]: Removing the + button

**Comment by tijlleenders** [comment]
The pull request contained too many files ( /dist folder and cypress test videos).
Merged the relevant changes manually with commit 6107022f5872f20ab1fc97d49c59d2b6604de9e9

**Pull request closed without merging**